queries/modHistory.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  collection,
+  getDocs,
+  limit,
+  orderBy,
+  query,
+  startAfter,
+} from 'firebase/firestore';
+import { db } from '../Firebase';
+
+const STRIDE = 3;
+
+export function getIQParams_ModHistory() {
+  return {
+    queryKey: ['modHistory', 'AAAAAAAAAAAAAAAAHHHHHHHHHHHH'], // 2nd param doesn't matter here bc no parent collection for modHistory
```

Comment:
You dont need to have the aahhhh; the queryKey can be an array of any number of strings, arrays, or objects and thus simply `['modHistory']` suffices